OpenRA.Game/Game.cs

```diff
@@ -363,6 +363,10 @@ static void Initialize(Arguments args)
 				}
 			}
 
+			var kdeDesktopFile = args.GetValue("Engine.KDEDesktopFile", null);
```

Comment:
This shouldn't be part of the game engine. Put it in `OpenRA.LinuxLauncher` or similar.

Reply:
I wouldn't object on principle to introducing a new launcher project, but how would that actually work? We need to know the X11 window ID before we can set this, which is an internal implementation detail of the platform window, which is owned by the engine. The only other way I can think of to set this without the engine knowing would be to use a helper thread and a pile of reflection to dig around in the internal state, which would be fragile and prone to race conditions. Am I missing something obvious here?

Reply:
Get rid of the command line argument here and check for `$XDG_CURRENT_DESKTOP` to detect KDE instead.

Reply:
Right, using environment variables is indeed something obvious that I was missing 🤦

I will rewrite this to have the platform window look for an `OPENRA_DESKTOP_FILE` environment variable, following the precident of our existing `OPENRA_DISPLAY_SCALE`. I will then use an `XDG_CURRENT_DESKTOP == KDE` check to only define the X11 property when actually needed.